config/routes.rb

```diff
@@ -15,6 +15,9 @@
     resources :assessments
     resources :direct_assessments
     resources :indirect_assessments
```

Comment:
It's in use on the outcomes#show page to display the table of indirect assessments.